core/base/inc/RtypesImp.h

```diff
@@ -44,7 +44,7 @@ inline void operator delete(void*, ROOT::Internal::TOperatorNewHelper*) { }
 
 // The STL GenerateInitInstance are not unique and hence are declared static
 // (not accessible outside the dictionary and not linker error for duplicate)
-#if defined(__CINT__)
+#if defined(__CLING__)
```

Comment:
This one is interesting.  The #ifdef is literally `if we are in CINT workaround problem in CINT`, so the 'right change is actually to drop completely the `#ifdef __CINT__` part and keep only the `#else` part